Playgrounds/1a_I_See_No_Nil.playground/contents.swift

```diff
@@ -146,7 +146,7 @@ For example the same does not apply to other ‘containers’, such as **Array**
 
 Whereas the following is acceptable:
 
-    let x:Int? = 8 // result is: Optional<Int>.Some(8)
+    let x:Int? = 8 // result is: Optional<Int>.some(8)
 
 * * *
 
```

Comment:
Huh, interesting objection. Let's see what the future brings :-)
Until then I have added the optional comparison operators to the playground, and added some explanation in 1ada018949, surely can be refined.